go.sum

```diff
@@ -1009,6 +1031,8 @@ github.com/modern-go/reflect2 v0.0.0-20180320133207-05fbef0ca5da/go.mod h1:bx2lN
 github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742/go.mod h1:bx2lNnkwVCuqBIxFjflWJWanXIb3RllmbCylyMrvgv0=
 github.com/modern-go/reflect2 v1.0.1/go.mod h1:bx2lNnkwVCuqBIxFjflWJWanXIb3RllmbCylyMrvgv0=
 github.com/modern-go/reflect2 v1.0.2/go.mod h1:yWuevngMOJpCy52FWWMvUC8ws7m/LJsjYzDa0/r8luk=
+github.com/modocache/gover v0.0.0-20171022184752-b58185e213c5/go.mod h1:caMODM3PzxT8aQXRPkAt8xlV/e7d7w8GM5g0fa5F0D8=
+github.com/montanaflynn/stats v0.6.6/go.mod h1:etXPPgVO6n31NxCd9KQUMvCM+ve0ruNzt6R8Bnaayow=
```

Comment:
Didn't spot these last time, but based on the name, wanted to be sure there's (e.g.) no telemetry sent (https://github.com/montanaflynn/stats), but looks like this is just used for tests, and not in other code; https://cs.github.com/AzureAD/microsoft-authentication-library-for-go/blob/8abadc88c59f379550986df569396be5d18e3554/apps/tests/performance/performance_test.go?q=github.com%2Fmontanaflynn%2Fstats#L20

Reply:
no longer there after the rebase